Documentation/troubleshooting.rst

```diff
@@ -31,15 +31,21 @@ fails between endpoints across multiple nods.
 Troubleshooting steps:
 ~~~~~~~~~~~~~~~~~~~~~~
 
-1. Run `cilium monitor` on the node of the source and destination endpoint.
+1. Run `cilium-health status` on the node of the source and destination
+   endpoint. It should describe the connectivity from that node to other
+   nodes in the cluster, and to a simulated endpoint on each other node. If
+   the command does not describe the status of the other node, there may be an
+   issue with the KV-Store.
+
+2. Run `cilium monitor` on the node of the source and destination endpoint.
    Look for packet drops.
 
 When running in :ref:`arch_overlay` mode:
 
-2. Run `cilium bpf tunnel list` and verify that each Cilium node is aware of
+3. Run `cilium bpf tunnel list` and verify that each Cilium node is aware of
    the other nodes in the cluster.  If not, check the logfile for errors.
 
-3. If nodes are being populated correctly, run `tcpdump -n -i cilium_vxlan` on
+4. If nodes are being populated correctly, run `tcpdump -n -i cilium_vxlan` on
```

Comment:
Is this interface always created even if we are not in overlay mode? Not sure if that has to be specified here or not.

Reply:
This section is for the overlay case only (line 43), although it isn't terribly apparent when skimming. I'm not sure how to make it clearer that there are two mutually exclusive sets of instructions :/

Reply:
@raybejjani yeah, that is my concern. Maybe we can have a list for if in overlay or if not? For now, I think this is fine.

Reply:
I suppose since the two cases are wholly separate a full set of instructions for each isn't bad. We could try to use this box thing http://docs.cilium.io/en/latest/policy/language/#simple-ingress-allow to make the separation very clear.
In any case, not a blocker today :)